recipes/pycairo/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set version = "1.10.0" %}
+{% set checksum = "9aa4078e7eb5be583aeabbe8d87172797717f95e8c4338f0d4a17b683a7253be" %}
+{% set checktype = "sha256" %}
+
+package:
+    name: pycairo
+    version: {{ version }}
+
+source:
+    fn: pycairo-{{ version }}.tar.bz2
+    url: https://cairographics.org/releases/pycairo-{{ version }}.tar.bz2
+    {{ checktype }}: {{ checksum }}
+
+build:
+    number: 0
+    skip: true                  # [win or py2k or py31 or py35]
```

Comment:
conda/conda-forge currently doesn't supply Python 3.1/3.2/3.3 versions and therefore I'm not sure if their selectors exist. I would probably just change this to `# [win or not py34]`